meilisearch-lib/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "meilisearch-lib"
-version = "0.25.0"
+version = "0.25.2"
```

Comment:
Why move from 0.25.0 to 0.25.2 here?

Reply:
because we are supposed to be in 0.25.1, we just forgot to update it in the last release

Reply:
haha, sorry, that's obvious!